.github/workflows/maven.yml

```diff
@@ -0,0 +1,26 @@
+# This workflow will build a Java project with Maven, and cache/restore any dependencies to improve the workflow execution time
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: Java CI with Maven
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
```

Comment:
Don't forget to add a **test** job. It is enough to run `mvn test` in order to run the unit tests.

Reply:
Fixed that